src/components/School.story.vue

```diff
@@ -0,0 +1,30 @@
+<script setup>
+import SchoolItem from './school/School.vue'
+import schools from './school.json'
+</script>
+<template>
+  <Story
+    title="Formations"
+    group="top1"
+    icon="mdi:school"
+    :layout="{ type: 'grid', width: '100%' }"
+  >
+    <Variant v-for="school in schools" :key="school.id" :title="school.name">
+      <SchoolItem
+        :city="school.city"
+        :date-from="school.dateFrom"
+        :date-to="school.dateTo"
+      >
+        <template #title>
+          <h2 class="htw-text-primary-500">{{ school.degreeName }}</h2>
+        </template>
+        <template #description>
```

Comment:
```suggestion
        </template>
        
        <template #description>
```

On hesite pas a faire des sautes de lignes la ou on sent que ce serait plus lisible ou beau

Reply:
genre pour separer des blocs

Reply:
Mais on en abuse pas non plus